docs/Configuration.md

```diff
@@ -252,34 +253,53 @@ The ZONEMASTER section has several keys :
 
 ### max_zonemaster_execution_time
 
-Time in seconds before reporting an unfinished test as failed. Default
-value: `600`.
+An integer.
+Time in seconds before reporting an unfinished test as failed.
+Default value: `600`.
 
 ### maximal_number_of_retries
 
+An integer.
 Number of time a test is allowed to be run again if unfinished after
-`max_zonemaster_execution_time`. Default value: `0`.
+`max_zonemaster_execution_time`.
+Default value: `0`.
+
+This option is experimental and all edge cases are not fully tested.
+Do not use it (keep the default value "0"), or use it with care.
```

Comment:
This comment is not directly for this PR, but just a note. We should not keep things as "experimental" forever.

Reply:
I created #750 to stabilize or remove it.